File: Practice Arrays/codigo_cs/6Running Comparison.cs
using System;
using System.Linq;

public class Test6
{
	public static void HappyParticipants()
	{
		// your code goes here
        int t = int.Parse(Console.ReadLine());
        for (int _ = 0; _ < t; _++)
        {
            int n = int.Parse(Console.ReadLine());
            int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
            int[] b = Console.ReadLine().Split().Select(int.Parse).ToArray();
            int count = 0;
            for (int i = 0; i < n; i++)
            {
                if (b[i] <= 2 * a[i] && a[i] <= 2 * b[i])
                {
                    count++;
                }
            }
            Console.WriteLine($"count: {count}");
        }
	}
}

File: Practice Arrays/6Running Comparison.py
t = int(input())

while t > 0:
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))
    t -= 1
    # Your code goes here
    count=0
    for i in range(n):
        # alice is happy
        if b[i]<=2*a[i] and a[i]<=2*b[i]:
            count+=1
    print("count",count)
